legend-sdlc-generation-service-maven-plugin/src/main/java/org/finos/legend/sdlc/generation/service/ServicesGenerationMojo.java

```diff
@@ -209,8 +210,9 @@ private static ResolvedServicesSpecification resolveServicesSpecification(Servic
         {
             try (EntityLoader directoriesLoader = EntityLoader.newEntityLoader(servicesSpec.directories))
             {
+                EntityToPureConverter converter = new EntityToPureConverter();
                 servicePaths = directoriesLoader.getAllEntities()
-                        .filter(e -> "meta::legend::service::metamodel::Service".equals(e.getClassifierPath()))
+                        .filter(e -> converter.fromEntityIfPossible(e).isPresent() && converter.fromEntityIfPossible(e).get() instanceof Service)
```

Comment:
There should be a way to do this without converting it twice, like maybe:
```suggestion
                        .filter(e -> converter.fromEntityIfPossible(e).filter(s -> s instanceof Service).isPresent())
```